bossdata/meta.py

```diff
@@ -98,36 +100,74 @@ def create_meta_lite(sp_all_path, db_path, verbose=True):
             to be a gzipped ASCII data file.
         db_path(str): Local path where the corresponding sqlite3 database will be written.
     """
+
+    # Hardcode to what I think is (quite) conservative value; smaller == slower.
+    # At 50k see memory peak ~2 GB, incldes holding ~750 MB of source in memory.
```

Comment:
I am confused by these memory sizes: the input file is only ~115Mb and the final sql file is ~470Mb.  How does this add up to 750Mb in memory at once?